freqtrade/plot/plotting.py

```diff
@@ -614,7 +615,7 @@ def store_plot_file(fig, filename: str, directory: Path, auto_open: bool = False
 
     _filename = directory.joinpath(filename)
     plot(fig, filename=str(_filename),
-         auto_open=auto_open)
+         auto_open=auto_open, include_plotlyjs=include_plotlyjs)
```

Comment:
we're intentionally importing `plot()` from `plotly.offline` - so we avoid any connection to plotly. 

that got several reasons (mainly around user's privacy) - so i think this setting is counter-productive to the approach taken to this.